tests/test_project.py

```diff
@@ -805,8 +805,9 @@ def test_schema(self):
         for i in range(10):
             self.project.open_job(
                 {
-                    "const": 0,
+                    "const1": 0,
                     "const2": {"const3": 0},
+                    "const4": {},
```

Comment:
This edge case was being missed until I added this test: https://github.com/glotzerlab/signac/blob/257c93169709039ef4ca54e8359aace31c62b0c8/signac/contrib/utility.py#L339-L340